cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -80,6 +82,9 @@ func init() {
 	if *connectionTimeout != 0 {
 		klog.Warningf("Warning: option -connection-timeout is deprecated and has no effect")
 	}
+	if *provisioner != "" {
```

Comment:
A bit confused why this section with `provisioner` (as well as the declaration above) is showing up as new code given it already exists in master and nothing has changed about it.